src/modules/vendor/pages/ni/config/ActivityProfile.vue

```diff
@@ -11,16 +11,32 @@
       </template>
     </ni-profile-header>
     <div class="row">
-      <ni-card-container class="col-md-4 col-xs-3" :cards="activity.cards" @add="openCardCreationModal"/>
+      <ni-card-container class="col-md-3 col-sm-4 col-xs-6" :cards="activity.cards" @add="openCardCreationModal"/>
     </div>
 
     <!-- Card creation modal -->
     <ni-modal v-model="cardCreationModal" @hide="resetCardCreationModal">
       <template slot="title">
         Créer une nouvelle <span class="text-weight-bold">carte</span>
       </template>
-      <ni-input in-modal v-model.trim="newCard.template" :error="$v.newCard.template.$error"
-        @blur="$v.newCard.template.$touch" required-field caption="Template" />
+      <h6 class="text-weight-bold">Cours</h6>
+      <div class="row">
+        <div @click="selectTemplateInModal(templateTypes[0].value)"
+          :class="getClassForTemplateInModal(templateTypes[0].value)">
+          <div class="text-weight-bold">Transition</div>
+        </div>
+        <div @click="selectTemplateInModal(templateTypes[1].value)"
+          :class="getClassForTemplateInModal(templateTypes[1].value)">
+          <div class="text-weight-bold">Titre</div>
+          <div>Text</div>
+          <div>Média</div>
+        </div>
+      </div>
```

Comment:
On ne pourrait pas faire un v-for sur template types plutot ?

Reply:
ok je fais un cmp

Reply:
Je ne sais pas si il y a besoin d'un composant mais tu peux juste mettre un v-for pour ne pas repeter le code

Reply:
done